### ISSUE TITLE
"Custom Column" with a `/` (slashes) in name will fail the query

### ISSUE THREAD

**Issue by flamber** [metabase/metabase#12305]
**Describe the bug**
Having a slash in the name of a Custom Column will make the query fail. This used to work on 0.34.3, so upgrading makes those questions fail.
**Workaround** is the change the slash to another symbol for the Custom Column, and set a slash in "Column title" if needed, in visualization Settings > Columns > (custom column) :gear:

**To Reproduce**
1. Custom question > Sample Dataset > Orders
2. Custom Column, formula `1 + 1`, name `TEST/test`
3. Visualize, results in error `Column "source.TEST/test" not found;`

**Information about your Metabase Installation:**
Metabase 0.35.1, various browser and backends.

**Additional context**
https://discourse.metabase.com/t/this-question-used-to-work-before-version-0-35-1/9626

**Comment by BenjaminGK** [comment]
@flamber This seems to not only be for custom questions but also if you have a pulse containing links in a table where the value contains '/'
Maybe not entirely the same but I get this error:
```
[17ae5802-b70c-45f6-8627-da05fec5b6c7] 2020-04-27T10:33:03+07:00 ERROR metabase.middleware.log GET /api/pulse/preview_card_info/1780 500 36.0 ms (7 DB calls)
{:message
"Unexpected character ('/' (code 47)): maybe a (non-standard) comment? (not recognized as one since Feature 'ALLOW_COMMENTS' not enabled for parser)\n at [Source: (StringReader); line: 1, column: 2]",
:type com.fasterxml.jackson.core.JsonParseException,
:stacktrace
["com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1804)"
"com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:669)"
"com.fasterxml.jackson.core.base.ParserMinimalBase._reportUnexpectedChar(ParserMinimalBase.java:567)"
"com.fasterxml.jackson.core.json.ReaderBasedJsonParser._skipComment(ReaderBasedJsonParser.java:2413)"
"com.fasterxml.jackson.core.json.ReaderBasedJsonParser._skipWSOrEnd2(ReaderBasedJsonParser.java:2388)"
"com.fasterxml.jackson.core.json.ReaderBasedJsonParser._skipWSOrEnd(ReaderBasedJsonParser.java:2339)"
"com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:646)"
"cheshire.parse$parse.invokeStatic(parse.clj:88)"
"cheshire.parse$parse.invoke(parse.clj:86)"
"cheshire.core$parse_string.invokeStatic(core.clj:206)"
"cheshire.core$parse_string.invoke(core.clj:192)"
"cheshire.core$parse_string.invokeStatic(core.clj:202)"
"cheshire.core$parse_string.invoke(core.clj:192)"
"clojure.core$comp$fn__5807.invoke(core.clj:2569)"
"clojure.core$comp$fn__5807.invoke(core.clj:2569)"
"medley.core$map_keys$fn__1769$fn__1770.invoke(core.cljc:103)"
"clojure.lang.PersistentArrayMap.kvreduce(PersistentArrayMap.java:377)"
"clojure.core$fn__8437.invokeStatic(core.clj:6845)"
"clojure.core$fn__8437.invoke(core.clj:6830)"
"clojure.core.protocols$fn__8167$G__8162__8176.invoke(protocols.clj:175)"
"clojure.core$reduce_kv.invokeStatic(core.clj:6856)"
"clojure.core$reduce_kv.invoke(core.clj:6847)"
"medley.core$reduce_map.invokeStatic(core.cljc:84)"
"medley.core$reduce_map.invoke(core.cljc:81)"
"medley.core$map_keys.invokeStatic(core.cljc:103)"
"medley.core$map_keys.invoke(core.cljc:100)"
"--> pulse.render.body$column_name.invokeStatic(body.clj:71)"
"pulse.render.body$column_name.invoke(body.clj:67)"
"pulse.render.body$query_results__GT_header_row$iter__32630__32634$fn__32635.invoke(body.clj:86)"
"pulse.render.table$render_table_head$iter__32478__32482$fn__32483.invoke(table.clj:59)"
"api.pulse$fn__64069.invokeStatic(pulse.clj:166)"
"api.pulse$fn__64069.invoke(pulse.clj:159)"
"middleware.auth$enforce_authentication$fn__66808.invoke(auth.clj:14)"
"routes$fn__68332$fn__68333.doInvoke(routes.clj:56)"
"middleware.exceptions$catch_uncaught_exceptions$fn__66907.invoke(exceptions.clj:97)"
"middleware.exceptions$catch_api_exceptions$fn__66904.invoke(exceptions.clj:85)"
"middleware.log$log_api_call$fn__68710$fn__68711.invoke(log.clj:197)"
"middleware.log$log_api_call$fn__68710.invoke(log.clj:191)"
"middleware.security$add_security_headers$fn__66870.invoke(security.clj:121)"
"middleware.json$wrap_json_body$fn__68413.invoke(json.clj:64)"
"middleware.json$wrap_streamed_json_response$fn__68431.invoke(json.clj:100)"
"middleware.misc$maybe_set_site_url$fn__68740.invoke(misc.clj:59)"
"middleware.session$bind_current_user$fn__63397$fn__63398.invoke(session.clj:209)"
"middleware.session$do_with_current_user.invokeStatic(session.clj:184)"
"middleware.session$do_with_current_user.invoke(session.clj:177)"
"middleware.session$bind_current_user$fn__63397.invoke(session.clj:208)"
"middleware.session$wrap_current_user_id$fn__63382.invoke(session.clj:161)"
"middleware.session$wrap_session_id$fn__63326.invoke(session.clj:125)"
"middleware.auth$wrap_api_key$fn__66816.invoke(auth.clj:27)"
"middleware.misc$bind_user_locale$fn__68745.invoke(misc.clj:78)"
"middleware.misc$add_content_type$fn__68726.invoke(misc.clj:29)"
"middleware.misc$disable_streaming_buffering$fn__68754.invoke(misc.clj:94)"
"server$async_proxy_handler$fn__68488.invoke(server.clj:70)"
"server.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a.handle(Unknown Source)"]}
```

**Comment by flamber** [comment]
@BenjaminGK Are you querying MongoDB? Then you might be seeing #11786.
I cannot reproduce with URLs returned in result sets from MariaDB and Postgres. Can you provide a way to reproduce this?

**Comment by BenjaminGK** [comment]
I am using MySQL but after deleting the question and creating a new one it got solved so not entirely sure if this was just a one off

**Comment by robdaemon** [comment]
Fix merged, targeted for 0.35.4 release.

**Comment by flamber** [comment]
This didn't get into 0.35.4, but it will be in 0.36.0
